examples/pytorch_lightning/msn.py

```diff
@@ -90,11 +91,11 @@ def configure_optimizers(self):
 pascal_voc = torchvision.datasets.VOCDetection(
     "datasets/pascal_voc", download=True, target_transform=lambda t: 0
 )
-dataset = LightlyDataset.from_torch_dataset(pascal_voc)
+dataset = LightlyDataset.from_torch_dataset(pascal_voc, transform=MSNTransform())
```

Comment:
In the pytorch example you set `MSNTransform(384)` but here you don't. What's the difference?

Reply:
Here it was already set. The 384 wasn't in the transform but in the linear layer. It was forgotten and also the old implementation was not working.

Reply:
Ah you're right.